### ISSUE TITLE
HuggingFace model class

### ISSUE THREAD

**Issue by lflage** [lflage/OpenFActScore#6]
In order to publish this we might make it more modular that is

- [ ] Implementing HFModel class
- [ ] Using logits for all open models to judge `supported` for AFV
    - Find position for True and False in vocab model, if True or False tokens are bigger than 1 (i.e. composed of sub word tokens) ignore and parse the generation.
- [ ] Importing and applying correct chat templates
    - Where to import Templates from
    - How to remove the template tokens properly

**Comment by lflage** [comment]
Solved in f211ac2f97602db26f94933c0ae9e23233685085
